PowerEditor/src/ScintillaComponent/AutoCompletion.h

```diff
@@ -21,6 +21,7 @@
 #include "tinyxml.h"
 
 const size_t tagMaxLen = 256;
+const auto FUNC_IMG_ID = 1000;

```

Comment:
Seems this could be moved to the cpp file as just used there. This is likely also the case for tagMaxLen. This has the advantage to be not visible for all who include AutoCompletion.h and within the global namespace.

constexpr for strings is not available before c++20 (https://www.modernescpp.com/index.php/constexpr-vector-and-string-in-c-20)

Reply:
moved the FUNC_IMG_ID to `.cpp` file.  The `tagMaxLen` was not added by this PR so I'm leaving it there for now.